src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -177,6 +179,27 @@ class CollapsibleCourseRun extends React.Component {
     return 'Submit Run for Review';
   }
 
+  formatDate(date) {
+    if (date) {
+      if (date.includes('T') &&
+        date.split('T')[0].match(/(19|20)\d{2}-(0\d{1}|1[0-2])-([0-2]\d{1}|3[0-1])/)) {
```

Comment:
I think all of these regexes should be written into a constants file with more clear names.